tests/lib/external/snapd-testing-tools/utils/spread-shellcheck

```diff
@@ -117,6 +117,16 @@ def checksection(data, env: Dict[str, str]):
     # shellcheck knows about that
     script_data = []
     script_data.append('set -e')
+    script_data.append('export SPREAD_BACKEND=placeholder')
+    script_data.append('export SPREAD_BACKENDS=placeholder,list')
```

Comment:
I think this is a whitespace sperated list: https://github.com/snapcore/spread/blob/ded9133cdbceaf01f8a1c9decf6ff9ea56e194d6/spread/project.go#L1012
```suggestion
    script_data.append('export SPREAD_BACKENDS="list placeholder"')
```